vms/platformvm/block/executor/standard_block_test.go

```diff
@@ -474,12 +474,11 @@ func TestBanffStandardBlockUpdateStakers(t *testing.T) {
 			},
 		},
 		{
-			description:   "advance time to staker5 end",
+			description:   "advance time to staker5 start",
 			stakers:       []staker{staker1, staker2, staker3, staker4, staker5},
 			advanceTimeTo: []time.Time{staker1.startTime, staker2.startTime, staker3.startTime, staker5.startTime},
 			expectedStakers: map[ids.NodeID]stakerStatus{
 				staker1.nodeID: current,
-				staker2.nodeID: current,
```

Comment:
@dhrubabasu another instance of the same brittleness we have in `TestBanffProposalBlockUpdateStakers` test. Staker2 should be evicted here, so not to appear among expected stakers. It is not evicted since the test does not generate the proposalBlockto do that. For the time being i just drop staker2 from the expected stakers. Will make another PR to fix this test.